marlowe-dashboard-client/src/Marlowe/PAB.purs

```diff
@@ -23,57 +19,9 @@ import Foreign.Class (class Encode, class Decode)
-import Wallet.Emulator.Wallet (Wallet) as Back
-import Wallet.Types (ContractInstanceId) as Back
 
-{-
-Marlowe requires three Plutus "contracts" to run in the PAB, which we refer to here as "apps" so as
-to avoid confusion with *Marlowe* contracts:
-
-1. The `MarloweApp`. This is used to:
-   - "create" Marlowe contracts (i.e. to install them on the blockchain and distribute the role tokens)
-   - "apply-inputs" to a Marlowe contract (i.e. perform an action to move that contract forward)
-   - "redeem" payments made to a role token
-2. The `WalletCompanion`. Every wallet has one instance of this app installed in the PAB. It listens
-   continuously for payments of role tokens to that wallet, and updates its observable state with an
-   array of `(MarloweParams, MarloweData)`, one pair for each role token, hence one for each Marlowe
-   contract for which this wallet has a role.
-3. The `MarloweFollower`. Every wallet has zero or more istances of this app installed in the PAB. We
-   use this to "follow" a Marlowe contract. Once a contract has been "created", and we have its
-   `MarloweParams`, we use an instance of the `MarloweFollower` to track its history and status. Once
-   we have called the "follow" endpoint of this app (passing it the `MarloweParams`), its observable
-   state will tell us everything we need to know about the contract, and will be updated when it changes.
-
-Note: to use this type directly in the `marlowe-pab` (the PAB bundled with the Marlowe apps), these
-names need to match those in `marlowe/pab/Main.hs`. To use the ContractExe type with the paths to the
-executables (with the PAB *without* the Marlowe apps bundled), we can call them whatever we like.
--}
-data PlutusApp
-  = MarloweApp
-  | WalletCompanion
-  | MarloweFollower
-
-derive instance eqPlutusApp :: Eq PlutusApp
-
-derive instance genericPlutusApp :: Generic PlutusApp _
-
-instance encodePlutusApp :: Encode PlutusApp where
-  encode value = genericEncode defaultOptions value
-
-instance decodePlutusApp :: Decode PlutusApp where
-  decode value = genericDecode defaultOptions value
-
-instance plutusAppContractActivationId :: ContractActivationId PlutusApp where
-  activateContract = defaultActivateContract
-  deactivateContract = defaultDeactivateContract
-  getContractInstanceClientState = defaultGetContractInstanceClientState
-  invokeEndpoint = defaultInvokeEndpoint
-  getWalletContractInstances = defaultGetWalletContractInstances
-  getAllContractInstances = defaultGetAllContractInstances
-  getContractDefinitions = defaultGetContractDefinitions
-
--- In the PAB, transactions have a fixed cost of 10 lovelace; in the real node,transaction fees will vary,
--- but this may still serve as a good approximation
+-- In the Marlowe PAB, transactions have a fixed cost of 10 lovelace; in the real node, transaction
+-- fees will vary, but this will serve as an approximation for now.
 transactionFee :: BigInteger
 transactionFee = fromInt 10
```

Comment:
Quick note, but unrelated to this PR. Fees are now configurable in the PAB given the configuration file. Maybe this needs to be the same value as the one in the config file?

Reply:
yes, it does (and is) 👍

Reply:
What I meant was: shouldn't this value come directly from the config file instead of hardcoding it here? If someone uses a different `FeeConfig` in the config file, he'll also have to manually change the hardcoded value here, no?

Reply:
Hmmm... I don't know how one would do that. I mean, just reading from the config file might work in development, but in production the config file isn't used. In any case I'm not too worried, as this is temporary - ultimately it should all be running on the real node with real fees.